packages/sdk/python/src/relayfile/types.py

```
@@ -267,6 +283,68 @@ class EventFeedResponse:
+class DigestSection:
+    provider: str
+    bullets: list[DigestBullet]
+
+
+class DigestContext(Protocol):
+    provider: str
+    window: DigestWindow
+
+    def change_events(
+        self,
+        filter: dict[str, list[str]] | None = None,
```

Ruff (0.15.12)

[error] Function argument `filter` is shadowing a Python builtin

(A002)